src/pages/Liquidity.jsx

```diff
@@ -11,6 +11,10 @@ const handleClick = () => {
     + 'outputCurrency=0x0327112423F3A68efdF1fcF402F6c5CB9f7C33fd';
 };
 
+const handleClickBalancer = () => {
```

Comment:
Can we use normal links?

Reply:
It was like this for the other button so I did the same

Reply:
Ok makes perfect sense. We'll optimise for SEO at some point